test/test.py

```diff
@@ -0,0 +1,8 @@
+from git_install_test_distribution import __version__
+
+def main():
+    print("Was able to het version of git_install_test_distribution:",__version__)
```

Comment:
typo! :D

Reply:
Fixed in [9861bd3](https://github.com/s-weigand/uv-windows-docker/pull/8/commits/9861bd34f32279e67889f7bfb66872548fb9d309)